### PR TITLE
Button Click

### PR DESCRIPTION
**Pull request by zerox1990**
Tried to make it like MenuPlayer.

### CODE REVIEW

**Comment by dannyflax** [comment]
Almost, but that's not quite what I was talking about.

What you need to do is put the public audiosource variable in MasterMonoBehaviour, then call a method in MasterMonoBehaviour to play the sound. This way you don't need to make those changes in the editor *every* time you have a button.

**Comment by zerox1990** [comment]
OK, placed public AudioSource var in MasterMono and made Click method that simply plays it.

File: SGoopas/Assets/Scripts/Game/Triggers/ButtonInteractTrigger.cs [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+ */
 public class ButtonInteractTrigger : InteractTrigger {
     private Vector3 originalScale;
-    private bool triggerOn = false;
-
+    private bool triggerOn = false;
+
     public override void Interact() {
         base.Interact();
         triggerOn = !triggerOn;
```
Comment (dannyflax) [inline SGoopas/Assets/Scripts/Game/Triggers/ButtonInteractTrigger.cs]: you have to call the method for it to play the sound 😝

**Comment by zerox1990** [comment]
I think I got it,

**Pull request closed without merging**